frontend/tests/controllers/RunDetailsTest.php

```diff
@@ -361,4 +361,97 @@ public function testUserInContest(
             $acRunData['details']['runs'][0]['contest_alias']
         );
     }
+
+    /**
+     * A PHPUnit data provider for the test with valid show_diff values.
+     *
+     * @return list<list<string>>
```

Comment:
esta anotación no debería ser

```suggestion
     * @return list<array{0: string, 1: array<string, array<string, string>>}>
```

?

Reply:
Ah, si. Olvidé cambiar el tipo cuando agregué el segundo índice del array